DataConnectors/AWS-S3/CloudWatchLanbdaFunction.py

```diff
@@ -0,0 +1,59 @@
+
+import boto3
+import json
+import csv
+import time
+import pandas as pd
+from datetime import datetime
+
+logs = boto3.client('logs')
+s3 = boto3.resource('s3')
+
+
+# Please set the following parameters:
+log_group_name = "" # Please enter log group name
+log_stream_name = "" # Please enter log stream name
+bucket_name = "" # Please enter bucket name
+bucket_prefix = "" # Please enter bucket prefix that ends with '/' , if no such, leave empty
+output_file_name = "" # Please change to desired name
+start_time = datetime(2022,12,15,6,40) # Please enter start time for exporting logs (year, month, day, hour, minutes) pay attention to time differences, here it should be UTC time
+end_time = datetime(2022,12,15,6,58) # Please enter end time for exporting logs (year, month, day, hour, minutes) pay attention to time differences, here it should be UTC time
+
+def lambda_handler(event, context):
```

Comment:
What is event or context?
"""
param1: events-
param2: context- 
"""

Reply:
![image](https://user-images.githubusercontent.com/98688758/213400053-b75de7a5-f6b3-4668-a11f-df60d66f70e7.png)

Reply:
I meant adding the built in function documentation python has for functions. 
It looks like this:
"""
param1: events-
param2: context-
"""
In there you can provide a oneliner explaining the parms so you don't have to go to the docs in order to understand.
An example can be: